tasks/main.yml

```diff
@@ -4,7 +4,7 @@
   when: nodejs_install_method == "package"
 
 - name: node.js | Check if the node version is already installed
-  command: node --version
+  shell: if hash node 2> /dev/null; then echo $(node --version); fi
```

Comment:
@bdossantos tried your version but for some reason it still executed the second part....